tests/functional/aws-node-sdk/test/object/post.js

```diff
@@ -0,0 +1,1621 @@
+// const AWS = require('aws-sdk');
+
+const xml2js = require('xml2js');
+const axios = require('axios');
+const crypto = require('crypto');
+const FormData = require('form-data');
```

Comment:
This dependency as well does not appear on `package.json`.
This is present only because it's a dependency of a dependency of a dependency, and is an old version.

Make sure to install dependencies you want to use, with latest version if possible (in `devDependencies` for tests)

```
> npm ls form-data
s3@7.70.47 /home/mickael/scality/cloudserver
└─┬ azure-storage@2.10.7
  └─┬ request@2.88.2
    └── form-data@2.3.3

> npm why form-data
form-data@2.3.3
node_modules/form-data
  form-data@"~2.3.2" from request@2.88.2
  node_modules/request
    request@"^2.86.0" from azure-storage@2.10.7
    node_modules/azure-storage
      azure-storage@"^2.1.0" from the root project
      azure-storage@"~2.10.7" from arsenal@7.70.29
      node_modules/arsenal
        arsenal@"git+https://github.com/scality/arsenal#4ef5748c028619edff10d6d38b21df43c8d63d88" from the root project
      azure-storage@"~2.10.7" from arsenal@7.10.46
      node_modules/bucketclient/node_modules/arsenal
      azure-storage@"~2.10.7" from arsenal@7.10.46
      node_modules/utapi/node_modules/arsenal
      azure-storage@"~2.10.7" from arsenal@7.10.46
      node_modules/vaultclient/node_modules/arsenal
    request@"^2.79.0" from google-auto-auth@0.9.7
    node_modules/google-auto-auth
      google-auto-auth@"^0.9.1" from the root project
    request@"^2.88.0" from oas-tools@2.2.2
    node_modules/oas-tools
      oas-tools@"^2.2.2" from utapi@7.70.4
      node_modules/utapi
        utapi@"git+https://github.com/scality/utapi#7.70.4" from the root project
```